README.md

```diff
@@ -59,24 +59,27 @@ a part of `site.yml`'s run. Use the `debug` option to get more insight to the pr
-| `http-listen-string`     | `"0.0.0.0:31836"`                     | Address/port the service will listen on. Use `127.0.0.1:31386` to lock down the UI.     |          |
-| `http-proto`             | `https`                               | Modify to "http" if necessary                                                           |          |
-| `http-user`              | `""`                                  | Username for HTTP Basic Auth                                                            |          |
-| `http-pass`              | `""`                                  | Password for HTTP basic Auth                                                            |          |
-| `http-url`               | `""`                                  | HTTP Url to find the Ansible tarball                                                    | yes      |
-| `log-dir`                | `"/var/log/ansible-puller"`           | Log directory (must exist)                                                              |          |
-| `ansible-dir`            | `""`                                  | Path in the pulled tarball to cd into before ansible commands - usually ansible.cfg dir |          |
-| `ansible-playbook`       | `"site.yml"`                          | The playbook that will be run  - relative to ansible-dir                                |          |
-| `ansible-inventory`      | `[]`                                  | List of inventories to operate on - relative to ansible-dir                             |          |
-| `venv-python`            | `"/usr/bin/python3"`                  | Path to the python version you are using for Ansible                                    |          |
-| `venv-path`              | `"/root/.virtualenvs/ansible_puller"` | Path to where the virtualenv will be created                                            |          |
-| `venv-requirements-file` | `"requirements.txt"`                  | Path to the python requirements file to populate the virtual environment                |          |
-| `sleep`                  | `30`                                  | How often to trigger run events in minutes                                              |          |
-| `start-disabled`         | `false`                               | Whether or not to start with Ansbile disabled (good for debugging)                      |          |
-| `debug`                  | `false`                               | Whether or not to start in debug mode                                                   |          |
-| `once`                   | `false`                               | Only run the configured playbook once and then stop                                     |          |
+| Config Option            | Default                               | Description                                                                             |
```

Comment:
The "Required" column was removed because nothing is required anymore

Reply:
Either "http-url" or "s3-obj" is required though, right? Maybe we can add that in the Description section?

Reply:
Done!